extensions/ql-vscode/src/cli.ts

```diff
@@ -66,13 +66,6 @@ export interface SourceInfo {
   sourceLocationPrefix: string;
 }
 
-/**
- * The expected output of `codeql resolve qlpacks`.
- */
-export interface ResolvedQLPacks {
-  [index: string]: string[];
-}
-
```

Comment:
Seems to be used in `src/qlpack-discovery.ts`, actually

Reply:
Oops. Fixed.